From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect struct cstate
Patch-mainline: never, kabi
References: kabi

In 4.4.129, commit 460439418f1c720d87baa55ed4c965fc9275576d (slip: Check
if rstate is initialized before uncompressing), upstream commit
3f01ddb962dc506916c243f9524e8bef97119b77 add a bool into struct cstate.
It made the kABI checker to complain.

Given the structure is internal to slhc, just hide it from the kABI
checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/net/slhc_vj.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/net/slhc_vj.h
+++ b/include/net/slhc_vj.h
@@ -127,7 +127,9 @@ typedef __u32 int32;
  */
 struct cstate {
 	byte_t	cs_this;	/* connection id number (xmit) */
+#ifndef __GENKSYMS__
 	bool	initialized;	/* true if initialized */
+#endif
 	struct cstate *next;	/* next in ring (xmit) */
 	struct iphdr cs_ip;	/* ip/tcp hdr from most recent packet */
 	struct tcphdr cs_tcp;
